### ISSUE TITLE
Editor.load should return status

### ISSUE THREAD

**Issue by pachacamac** [newcat/baklavajs#155]
For example if I load an editor state and I have not registered a node type that is used, there are some warning logged: 

```
Node type TestNode is not registered
Could not find interface with id ni_162594158296227
```

but there's no way of easily checking if the loading was successful.

**Comment by newcat** [comment]
True! I have two ideas how to solve this, please tell me what you like more (could also do both actually):
1. Add an optional `throwOnError` parameter to the load function. In that case, instead of silently failing it will throw an error and fail to load completely
2. Make the `load` function return an array of errors. These errors could be either raw strings like what you mentioned above or objects like:
```
[
  { errorType: "missingNodeType", nodeType: "TestNode"},
  { errorType: "missingInterface", id: "ni_162594158296227" }
]
```

**Comment by pachacamac** [comment]
The second option is basically what I'm doing right now but checking it after the load step by step. So that would probably be my preferred way. I also like that the whole load doesn't fail if one node type is missing but others are available. So I'm just showing a warning message to the user which types are missing.

**Comment by newcat** [comment]
Implemented in [v1.9.1](https://github.com/newcat/baklavajs/releases/tag/v1.9.1)
